scrapy/utils/log.py

```diff
@@ -174,12 +175,11 @@ def log_scrapy_info(settings: Settings) -> None:
         "Scrapy %(version)s started (bot: %(bot)s)",
         {"version": scrapy.__version__, "bot": settings["BOT_NAME"]},
     )
-    versions = [
-        f"{name} {version}"
-        for name, version in scrapy_components_versions()
-        if name != "Scrapy"
-    ]
-    logger.info("Versions: %(versions)s", {"versions": ", ".join(versions)})
+    software = settings.getlist("LOG_VERSIONS")
```

Comment:
Looks good! Just wondering if this variable should be in plural.

Reply:
”software” is plural, singular would be “pieces of software”, “software components”, or similar. I went with “items” to keep things short.

Reply:
TIL, thank you @Gallaecio